packages/core/src/app/customer/CheckoutButtonContainer.tsx

```diff
@@ -54,8 +56,17 @@ const CheckoutButtonContainer: FunctionComponent<CheckoutButtonContainerProps &
         return null;
     }
 
+    const paypalCommerceIds = ['paypalcommerce', 'paypalcommercecredit', 'paypalcommercevenmo'];
+    const isPaypalCommerce = methodIds.some(id => paypalCommerceIds.includes(id));
```

Comment:
Future optimisation, we should be able to move these methods out of the component body. So that they are not re-created on every re render.

Reply:
I have moved it to the wrapper/map function.